sdk/communication/azure-communication-rooms/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
 # Release History
 
-## 1.0.0-beta.1 (2022-08-12)
+## 1.0.0-beta.2 (2022-08-12)
```

Comment:
I think you will want to leave 1.0.0-beta.1 there, and add 1.0.0-beta.2 above it?

Reply:
Since 1.0.0-beta.1 was never released public, did not add an entry.
Thoughts?

Reply:
1.0.0-beta.1 is a dud release that somehow got published in [Jan this year](https://dev.azure.com/azure-sdk/public/_artifacts/feed/azure-sdk-for-java/maven/com.azure%2Fazure-communication-rooms/overview/1.0.0-beta.1)

This `beta.2` version will essentially be the first publicly available version of our Java SDK package. Doesn't look great, but it's the best option we have available now

Reply:
Beta 1 was released publicly. It was during that release we found that there was already a beta.1 in the dev feed but this wasn't until after it was released to maven. Keep the beta.1 entry and add a new one for beta.2.

Reply:
Jim, please help me understand, 
My understanding was the beta.1 version got released to internal library and did not make it to public. As I could not find it in maven repository. 
Where can I find the beta1 version on public?

Reply:
Updated Readme to include history of beta.1

Thanks for the DM conversation team. Posting understanding here.

Though the devfeed build failed at [Pipelines - Run 20220812.1 logs (azure.com)](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1777383&view=logs&j=49259b2a-9f18-5945-e5de-00969e5597f4&t=fa97ad70-7b0d-54e1-14e4-7630c1c8441d) the beta.1 package is released to public maven repository - is this understanding right. Indexing will take time.

2. Since the devfeed build failed, the auto version increment PR did not come through. 

3. We have this PR to fix the version update [[ACS Rooms] Beta release version update by elavarasidc · Pull Request #30442 · Azure/azure-sdk-for-java (github.com)](https://github.com/Azure/azure-sdk-for-java/pull/30442)

And releasing beta.2 will also fix an update to readme and changelog. 

If any of the above understanding is not right, please correct me.